languages.yaml

```
@@ -101,6 +100,8 @@ it:
   leaders:
     - auanasgheps
     - xraver
+ja:
+  nativeName: japanese
```

The native name has to be in Japanese. 
```suggestion
  nativeName: 日本語
```